src/lib/date-picker/month-view.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  Component,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  Input,
+  EventEmitter,
+  Output,
+  AfterContentInit
+} from '@angular/core';
+import {coerceDateProperty} from '../core/coercion/date-property';
+import {MdCalendarCell} from './calendar-table';
+import {DateLocale} from './date-locale';
+
+
+const DAYS_PER_WEEK = 7;
```

Comment:
PI is always 3.1415... :p it makes the code more readable. The constant tells you why there's magic 7's floating around in the expression